tests/test_interpolation.py

```diff
@@ -250,6 +250,21 @@ def test_resolver_cache_2():
         OmegaConf.clear_resolvers()
 
 
+@pytest.mark.parametrize('resolver,name,key,result', [
+    (lambda *args: args, 'arg_list', "${my_resolver:cat, dog}",("cat", "dog")),
+    (lambda *args: args, 'escape_comma',"${my_resolver:cat\, do g}",("cat, do g",)),
+    (lambda *args: args, 'escape_whitespace',"${my_resolver:cat\, do g}",("cat, do g",)),
+    (lambda: 'zero', 'zero_arg',"${my_resolver:}",("zero")),
+])
```

Comment:
Not familiar with this syntex. Why do you need it here?

Reply:
I'm not sure what you mean? Would you prefer I passed in entire config dictionaries as opposed to pairs of `key, string_to_resolve`?

Reply:
oh, I was confused by the lambda being passed there, I get why you need this now.